core/src/main/java/org/frankframework/receivers/Receiver.java

```diff
@@ -980,8 +980,10 @@ public void moveInProcessToError(RawMessageWrapper<M> rawMessageWrapper, Map<Str
 				message = getListener().extractMessage(rawMessageWrapper, context);
 			}
 			throwEvent(RCV_MESSAGE_TO_ERRORSTORE_EVENT, message);
-			if (errorSender!=null) {
-				errorSender.sendMessageOrThrow(message, null);
+			if (errorSender != null) {
+				try(PipeLineSession session = new PipeLineSession(); Message senderResult = errorSender.sendMessageOrThrow(message, session)) {
```

Comment:
Heeft de receiver hier niet al een PipeLineSession die gebruikt kan worden hiervoor? Het `threadContext` argument is vaak een PipeLineSession, alleen de aanroep vanuit de PullingListenerContainer heeft op dat moment geen session.

Reply:
Ik zou de ThreadContext kunnen gebruiken, maar dit is nooit nodig geweest en ik wil niet nu mogelijk 'message is already closed' problemen gaan introduceren. Vandaar dat ik in de meeste gevallen voor de meer veilige 'nieuwe short-lived session' gekozen.